tests/scripts/set_psa_test_dependencies.py

```diff
@@ -22,6 +22,15 @@
 import re
 import sys
 
+CLASSIC_DEPENDENCIES = frozenset([
```

Comment:
I would prefer to be on the side of having missing dependencies rather than wrong dependencies and thus test cases failing rather than not running. Thus what about to rather list the classic dependencies that we should keep given the fact that generally speaking we are more likely to miss things in such table.

Reply:
I don't understand. Anything not listed here is kept. This can be anything the programmer who wrote the test case decided to add, including symbols that are specific to the test suite. It's impossible to list all the symbols that should stay. (In addition to being impractical even for the symbols from `config.h`, which include a lot of `MBEDTLS_X509_xxx` and `MBEDTLS_SSL_xxx`.)

Reply:
ok, it hadn't in mind that the mirror list would be such a nightmare.